src/frontends/pytorch/src/transforms/prim_list_construct_pad.cpp

```diff
@@ -0,0 +1,118 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "prim_list_construct_pad.hpp"
+
+#include "openvino/core/rt_info.hpp"
+#include "openvino/op/add.hpp"
+#include "openvino/op/broadcast.hpp"
+#include "openvino/op/concat.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/convert_like.hpp"
+#include "openvino/op/gather.hpp"
+#include "openvino/op/pad.hpp"
+#include "openvino/op/range.hpp"
+#include "openvino/op/shape_of.hpp"
+#include "openvino/op/subtract.hpp"
+#include "openvino/op/util/framework_node.hpp"
+#include "openvino/pass/pattern/matcher.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "pt_framework_node.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace pass {
+
+PrimListConstructPadReplacer::PrimListConstructPadReplacer() {
+    auto pad_op = ov::pass::pattern::wrap_type<ov::op::util::FrameworkNode>();
+    ov::matcher_pass_callback callback = [](ov::pass::pattern::Matcher& m) {
+        auto pad_op = cast_fw_node(m.get_match_root(), "aten::pad");
+        if (!pad_op) {
+            return false;
+        }
+        auto pt_node = std::dynamic_pointer_cast<PtFrameworkNode>(pad_op);
+        auto input_node = pad_op->input_value(0).get_node_shared_ptr();
+        auto padding = pad_op->input_value(1).get_node_shared_ptr();
+        auto pad_values = concat_list_construct(padding);
+        std::string mode = "constant";
+        auto minus_one = ov::op::v0::Constant::create(element::i64, Shape{}, {-1});
```

Comment:
please use the latest opset and `using namespace ov::opset10;` to avoid explicit ``ov::op::..

Reply:
added